### DISCUSSION TITLE
Table component - cell wrapping

### DISCUSSION THREAD

**Discussion by olton** [olton/Metro-UI-CSS#1715] [Q&A]
Discussion about a Table component

**Comment by BeckerIndustries** [comment]
Hello,

I have created a table

```html
       <div class="m-3">
          <div data-role="panel" data-title-caption="Link Database" data-collapsible="false" data-title-icon="<span class='mif-link'></span>" class="mt-4">
            <div class="row">
              <div class="cell-md-12 p-5">
                <table class="table striped table-border mt-4" id="example" data-role="table" data-pagination="true">
                  <thead>
                    <tr>
                      <th class="sortable-column sort-desc">Name</th>
                      <th>Description</th>
                      <th>URL</th>
                    </tr>
                  </thead>
                  <body>
                    <tr>
                      <td style="width: 1px"><a href="<?php echo $row['url']; ?>" target="_blank"><?php echo $row['title']; ?></a></td>
                      <td style="word-wrap: break-word; max-width: 100px;"><?php echo $row['description']; ?></td>
                      <td><?php echo $row['url']; ?></td>
                    </tr>
                  </tfoot>
                </table>
              </div>
            </div>
          </div>
        </div>
```

However, I want the #2 column to have a large amount of text that breaks after it reaches a certain size.  I am essentially making a table with links to several different systems.  And the description column may contain several sentences to describe the system.  I've tried max width and word-wrap with no change.

**Reply by olton** [reply]
> However, I want the #2 column to have a large amount of text that breaks after it reaches a certain size. I am essentially ?
> making a table with links to several different systems. And the description column may contain several sentences to describe 
> the system. I've tried max width and word-wrap with no change.

Use attribute `data-cell-wrapper="false"` for a table component.

**Comment by olton** [comment]
Please respect the community members' time, create examples on codepen.io so that they can be viewed in detail! And format your code!

**Reply by olton** [reply]
> My code was formated. That's githubs fault.

To format multi-line code you must use three back quotes and a hint
